code/datums/spells/lightning.dm

```diff
@@ -0,0 +1,89 @@
+/obj/effect/proc_holder/spell/targeted/lightning
+	name = "Lightning Bolt"
+	desc = "Throws a lightning bolt at the nearby enemy. Classic."
+	charge_type = "recharge"
+	charge_max	= 300
+	clothes_req = 1
+	invocation = "UN'LTD P'WAH!"
+	invocation_type = "shout"
+	range = 7
+	cooldown_min = 30
+	selection_type = "view"
+	random_target = 1
+	var/energy = 0
+	var/ready = 0
+
+/obj/effect/bolt
+	name = "Lightning bolt"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "lightning"
+	luminosity = 3
+
+/obj/effect/proc_holder/spell/targeted/lightning/Click()
+	if(!ready)
+		if(cast_check())
+			StartChargeup()
+	else
+		if(cast_check(skipcharge=1))
+			choose_targets()
+	return 1
+
+/obj/effect/proc_holder/spell/targeted/lightning/proc/StartChargeup(mob/user = usr)
+	ready = 1
+	user << "<span class='notice'>You start gathering the power.</span>"
+	//TODO: Add visual indictaor of charging
```

Comment:
I think a good visual indicator would be to add a glow to the caster, like with Cold Resistance. It could pulse quicker the longer you charge it.